draft-ietf-httpbis-digest-headers.md

```diff
@@ -278,6 +278,8 @@ A recipient MAY ignore any or all of the representation-data-digests in a Digest
 field. This allows the recipient to choose which digest-algorithm(s) to use for
 validation instead of verifying every received representation-data-digest.
 
+A recipient MUST ignore a representation-data-digest containing parameters.
```

Comment:
What does this _mean_? `representation-data-digest` above does not contain syntax for `parameters`, so how does a parser know if parameters are present?

Reply:
My comment was stuck pending. I think mnot and I have the same thought

Reply:
Thanks @mnot @LPardue ! 

The confusion comes from the fact that the old spec allowed for parameters(even if they were not presenti in the syntax) while this one does not.

I wanted to provide guidance in how to manage a Digest header sent by an old implementation: which is the correct way to address this issue?

Hyp 1: add the parameter definition to the ABNF and explicit that they SHOULD|MUST be ignored, while the checksum should be processed

Hyp 2: add the parameter definition to the ABNF and explicit that if a `representation-data-digest` contains parameter *only this checksum should be ignored*

Hyp 3: the presence of a parameter in any `representation-data-digest` will invalidate the syntax of the whole Digest value.


Thanks for your support, R.